server_addon/substancepainter/server/settings/load_plugins.py

```diff
@@ -0,0 +1,122 @@
+        title="Document Resolution",
+        description=("Set texture resolution when "
+                     "creating new project.")
+    )
+    import_cameras: bool = SettingsField(
+        True, title="Import Cameras",
+        description="Import cameras from the mesh file.")
+    normal_map_format: str = SettingsField(
+        "DirectX", enum_resolver=normal_map_format_enum,
+        title="Normal Map Format",
+        description=("Set normal map format when "
+                     "creating new project.")
+    )
+    project_workflow: str = SettingsField(
+        "Default", enum_resolver=uv_workflow_enum,
+        title="UV Tile Settings",
+        description=("Set UV workflow when "
+                     "creating new project.")
+    )
+    tangent_space_mode: str = SettingsField(
+        "PerFragment", enum_resolver=tangent_space_enum,
+        title="Tangent Space",
+        description=("An option to compute tangent space "
+                     "when creating new project.")
+    )
+    preserve_strokes: bool = SettingsField(
+        True, title="Preserve Strokes",
+        description=("Preserve strokes positions on mesh.\n"
+                     "(only relevant when loading into "
+                     "existing project)")
+    )
+
+
+class ProjectTemplateSettingModel(BaseSettingsModel):
+    project_templates: list[ProjectTemplatesModel] = SettingsField(
+        default_factory=ProjectTemplatesModel,
+        title="Project Templates"
+    )
+
+
+class LoadersModel(BaseSettingsModel):
+    SubstanceLoadProjectMesh: ProjectTemplateSettingModel = SettingsField(
+        default_factory=ProjectTemplateSettingModel,
+        title="Load Mesh"
+    )
+
+
+DEFAULT_LOADER_SETTINGS = {
+    "SubstanceLoadProjectMesh": {
+        "project_templates": [
+            {
+                "name": "2K(Default)",
+                "default_texture_resolution": 2048,
+                "import_cameras": True,
+                "normal_map_format": "NormalMapFormat.DirectX",
+                "project_workflow": "ProjectWorkflow.Default",
+                "tangent_space_mode": "TangentSpace.PerFragment",
+                "preserve_strokes": True
+            },
+            {
+                "name": "2K(UV tile)",
+                "default_texture_resolution": 2048,
+                "import_cameras": True,
+                "normal_map_format": "NormalMapFormat.DirectX",
+                "project_workflow": "ProjectWorkflow.UVTile",
+                "tangent_space_mode": "TangentSpace.PerFragment",
+                "preserve_strokes": True
+            },
+            {
+                "name": "4K(Custom)",
+                "default_texture_resolution": 4096,
+                "import_cameras": True,
+                "normal_map_format": "NormalMapFormat.OpenGL",
+                "project_workflow": "ProjectWorkflow.UVTile",
+                "tangent_space_mode": "TangentSpace.PerFragment",
+                "preserve_strokes": True
```

Comment:
Should we still take the `import_cameras` and `preserve_strokes` out of the template @moonyuet ?

Reply:
Do you mean adding these two options into the Project Configuration Dialog and let the user choose?

Reply:
Yes - exactly. Since that came up before. especially because they seem artist choices - and `preserve_strokes` is really only crucial on updates based on what the artist wants to do?